flask_app/tasks.py

```diff
@@ -299,8 +307,11 @@ def vacuum():
 @needs_app_context
 def refresh_issue_trackers():
     trackers = db.session.query(Tracker)
+    issues_of_active_beams = {beam.issue.id for beam in _get_active_beams()}
     for tracker in trackers:
-        issues = db.session.query(Issue).filter_by(tracker_id=tracker.id)
+        issues = db.session.query(Issue).\
```

Comment:
This will probably work but the performance won't be good. You should join the beam. I think this will help:

https://www.postgresql.org/docs/8.1/functions-subquery.html